### PR TITLE
Add date sorting on tag hompepage

### PR DESCRIPTION
**Pull request by aguestuser**
as per review of #234 / #312  this AM, add sorting to tag homepage

### CODE REVIEW

File: app/views/tags/.tern-port [hunk 1]
```diff
@@ -0,0 +1 @@
+35497
```
Comment (aepyornis) [inline app/views/tags/.tern-port]: delete this? probably some editor thing 😆

**Comment by aguestuser** [comment]
alert, alert! forgot to style this! will do that real quick before merging!

**Comment by aepyornis** [comment]
sounds good!

**Comment by aguestuser** [comment]
@aepyornis okay! ready to merge! 

here are updated at date stylings:

tag-homepage-date-styling

here is a re-styling of empty list message views (mentioned in commit 22a2b2b):

tag-homepage-empty-styling

**Comment by aguestuser** [comment]
3... 2... 1... merging!

**Pull request merged**